Source/Core/VideoCommon/XFStructs.cpp

```diff
@@ -93,7 +93,7 @@ static void XFRegWritten(int transferSize, u32 baseAddress, DataReader src)
       break;
 
     case XFMEM_DUALTEX:
-      if (xfmem.dualTexTrans.enabled != (newValue & 1))
+      if (xfmem.dualTexTrans.enabled != ((newValue & 1) != 0))
```

Comment:
What happened here?

Reply:
In visual studio, it produces a signed-unsigned mismatch warning otherwise (which is then treated as an error).  That happens even if `(newValue & 1u)` or `u32(newValue & 1)` is used though `s32(newValue & 1)` works, as does the more natural `bool(newValue & 1)`.  The same also happens if `enabled.Value()` is used &mdash; I'm guessing it's because it's promoting the `bool` to an `int` and then getting mad when it's not unsigned.